services/essentials.yaml

```diff
@@ -39,27 +45,57 @@ spec:
     - groupId: org.jenkins-ci.plugins
       artifactId: junit
       version: 1.26.1
+    - groupId: org.jvnet.hudson.plugins
+      artifactId: favorite
+      version: 2.3.2
     - groupId: com.coravy.hudson.plugins.github
       artifactId: github
-      version: 1.29.2
+      version: 1.29.3
+    - groupId: org.jenkins-ci.plugins
+      artifactId: github-api
+      version: '1.92'
```

Comment:
Seems the YAML serializer is more clever than we might like: `x.y` is quoted as a string but `x.y.z` is not, even though for our purposes we know all `version`s ought to be loaded as strings. JavaScript :man_shrugging: